src/inline_string.rs

```diff
@@ -674,6 +683,19 @@ mod tests {
 
         assert_eq!(s.insert(0, 'a'), Err(NotEnoughSpaceError));
     }
+
+    #[test]
+    fn test_write() {
+        use fmt::{Error, Write};
+
+        let mut s = InlineString::new();
+
+        for _ in 0..INLINE_STRING_CAPACITY {
+            assert!(write!(&mut s, "a").is_ok());
+        }
```

Comment:
We should probably test that `s` contains the right contents now, too.

Reply:
You've got it.